examples/3d/lighting.rs

```diff
@@ -34,7 +36,7 @@ fn setup(
 
     // left wall
     let mut transform = Transform::from_xyz(2.5, 2.5, 0.0);
-    transform.rotate_z(std::f32::consts::FRAC_PI_2);
+    transform.rotate_z(TAU / 4.);
```

Comment:
This has less accuracy than FRAC_PI_2.

Reply:
I don't think the loss of precision is relevant to any of our current examples,
but I wanted to investigate the difference in precision so i tried this:
```rust
assert_eq!(
    std::f64::consts::TAU / 4.,
    std::f64::consts::FRAC_PI_2
);
```
This assertion does not fail however. Is this test too naive?

Reply:
I'd compare the `to_bits` outcome to be safe

Reply:
Same result with `to_bits`.